pybaseball/statcast_running.py

```diff
@@ -0,0 +1,24 @@
+import io
+from typing import Optional, Union
+
+import pandas as pd
+import requests
+
+from . import cache
+
+@cache.df_cache()
+def statcast_sprint_speed(year: int, min_opp: int = 10) -> pd.DataFrame:
+	url = f"https://baseballsavant.mlb.com/leaderboard/sprint_speed?year={year}&position=&team=&min={min_opp}&csv=true"
+	res = requests.get(url, timeout=None).content
+	data = pd.read_csv(io.StringIO(res.decode('utf-8')))
+	return data
+
+@cache.df_cache()
+def statcast_running_splits(year: int, min_opp: int = 5, split_type: str = "raw") -> pd.DataFrame:
+	splits = ["raw", "percent"]
+	if split_type not in splits:
+		raise ValueError(f"Not a valid split_type. Must be one of {', '.join(splits)}.")
```

Comment:
I would suggest that for just two options change this to a bool

Reply:
Ah yeah will update that